### PR TITLE
Fix builtin names

### PR DESCRIPTION
**Pull request by michaelpj**
When the printing of builtins was changed to match the constructor
names, we got some less-nice names as a result. This fixes that by
making the constructor names nicer.

Pre-submit checklist:
- Branch
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reviewer requested

Pre-merge checklist:
- [ ] Someone approved it
- [ ] Commits have useful messages
- [ ] Review clarifications made it into the code
- [ ] History is moderately tidy; or going to squash-merge

### CODE REVIEW

File: plutus-core/cost-model/budgeting-bench/Bench.hs [hunk 1]
```diff
@@ -320,7 +320,7 @@ benchNop3 gen =
    DivideInteger:     RemainderInteger, QuotientInteger, ModInteger
    LessThanInteger:   GreaterThanInteger
    LessThanEqInteger: GreaterThanEqInteger
-   LtByteString:      GtByteString
+   LessThanByteString:      GreaterThanByteString
```
Comment (effectfully) [inline plutus-core/cost-model/budgeting-bench/Bench.hs:323]: Do we care about the ruined formatting here?

**Review by effectfully — APPROVED** [review]

File: plutus-core/cost-model/budgeting-bench/Bench.hs (hunk 1 above)
Comment (michaelpj) [reply plutus-core/cost-model/budgeting-bench/Bench.hs:323]: I don't :)

Comment (kwxm) [reply plutus-core/cost-model/budgeting-bench/Bench.hs:323]: I'll probably notice this and fix it the next time I'm working on this file.

**Review by kwxm — APPROVED** [review]
Everything seems to be working fine now: thanks!  I'll merge this now.

**Comment by kwxm** [comment]
There was a failure because of this (mac-mini-1):

```
Generating inventory for haddock dir './marlowe-symbolic/html'
/private/tmp/nix-build-haddock-join.drv-0/.attr-0: line 62: /tmp/doc-index.json: Permission denied
builder for '/nix/store/4p03zg9bcbxwmps19rdc0n6njbfxm3m0-haddock-join.drv' failed with exit code 1
```

I'm going to disregard that and merge anyway.

**Pull request merged**